src/java/boa/compiler/SymbolTable.java

```diff
@@ -192,6 +192,7 @@
 
 		// set functions
 		globalFunctions.addFunction("contains", new BoaFunction(new BoaBool(), new BoaType[] { new BoaSet(new BoaTypeVar("V")), new BoaTypeVar("V") }, "${0}.contains(${1})"));
+		globalFunctions.addFunction("containsAll", new BoaFunction(new BoaBool(), new BoaType[] { new BoaSet(new BoaTypeVar("V")), new BoaSet(new BoaTypeVar("V"))}, "${0}.containsAll(${1})"));
```

Comment:
Should the function naming conventions be similar to current naming conventions used within Boa?